proto/proto-backplane-grpc/build.gradle

```diff
@@ -88,6 +90,18 @@ artifacts {
   cpp(layout.buildDirectory.dir('generated/source/proto/main/cpp')) {
     builtBy generateProtobuf
   }
+  html(layout.buildDirectory.dir('generated/source/proto/main/proto-doc/single-html')) {
+    builtBy generateProtobuf
+  }
+  html(layout.buildDirectory.dir('generated/source/proto/main/proto-doc/multi-html')) {
+    builtBy generateProtobuf
+  }
```

Comment:
I don't know if we can specify multiple artifacts like this for a single configuration.

I would probably just create a single new configuration `protoDocs` with the top level build directory:

```groovy
protoDocs(layout.buildDirectory.dir('generated/source/proto/main/proto-doc')) {
...
}
```

The other alternative is to create a separate configuration and artifact for each output.